content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -98,7 +98,7 @@ desiredReplicas = ceil[currentReplicas * ( currentMetricValue / desiredMetricVal
 
 For example, if the current metric value is `200m`, and the desired value
 is `100m`, the number of replicas will be doubled, since `200.0 / 100.0 ==
-2.0` If the current value is instead `50m`, we'll halve the number of
+2.0` If the current value is instead `50m`, we'll have the number of
```

Comment:
The original is actually correct! 

To _halve_ something = to take half of.

As such I'm going to close this, but thank you for catching this!

/close